CONTRIBUTING.md

```diff
@@ -46,6 +46,45 @@ Make sure you have the following dependencies installed first:
    mage -v
    ```
 
+## Local development with Grafana
```

Comment:
Local development with local Grafana?

Reply:
(avoided one "local" because of redundancy)